terraform/dotnet/k8s/deploy/resources/python-frontend-service-depl.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: python-sample-app-deployment-${TESTING_ID}
+  namespace: python-sample-app-namespace
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python-sample-app
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: python-sample-app
+      annotations:
+        instrumentation.opentelemetry.io/inject-python: "true"
+    spec:
+      containers:
+        - name: python-frontend
+          image: ${IMAGE}
+          args:
+            - "sh"
+            - "-c"
+            - "python3 manage.py migrate --noinput && python3 manage.py collectstatic --noinput && python3 manage.py runserver 0.0.0.0:8000 --noreload"
+          ports:
+            - containerPort: 8000
+          env:
+            - name: "OTEL_SERVICE_NAME"
+              value: "python-sample-application-${TESTING_ID}"
+            - name: "PYTHONPATH"
+              value: "/django_frontend_app"
+            - name: "DJANGO_SETTINGS_MODULE"
+              value: "django_frontend_service.settings"
+      imagePullSecrets:
+        - name: ecr-secret
```

Comment:
Why is everything named after Python?

Reply:
Temporarily remove all k8s file now, since not publish K8s for now, it's not ready